go/saltpackkeys/saltpackkeysmocks/saltpack_mock_pseudonym_resolver.go

```diff
@@ -0,0 +1,48 @@
+package saltpackkeysmocks
```

Comment:
Are you sure we need a separate package for this? I assume it's just for testing, right?